### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][Eclipse][Wipro]:Unable to access the scroll bar which is present in run on web app for containers window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1414]
**Repro Steps:**
1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Publish as Azure web app for containers.
4. "Run on web app for containers" dialog, tab up to use existing radio button which is present under "Web app for container" then by using arrow key select "create New" radio button then try to access the scroll bar

**Expected Result:**
Scroll bar should be accessible by using keyboard

**Actual Result:**
Scroll bar is not accessible by using keyboard

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-06
**OSBuild Version:** Version 1709(Os Build 16299.192)

![scroll bar is not accessible in run on web app for containers](https://user-images.githubusercontent.com/22956130/35716622-167701c0-0800-11e8-99d7-00dfdcb55f85.jpg)

**Comment by Eskibear** [comment]
It's part of Eclipse's UI, out of the toolkit's control.

**Comment by Eskibear** [comment]
it's an existing issue here https://bugs.eclipse.org/bugs/show_bug.cgi?id=87840

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749874
